connector/connect/server/pom.xml

```diff
@@ -155,6 +155,12 @@
       <version>${protobuf.version}</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
```

Comment:
Without this pr, the server module depended on `protobuf-java-util:3.19.2 `. Should `protobuf-java-util` and `protobuf-java` always use the same version?

Reply:
I removed the dependency and mvn build fails. But SBT is ok. Not sure where SBT gets the dependency from.
What shall we do? 

<img width="2263" alt="image" src="https://user-images.githubusercontent.com/502522/224126061-736648ee-da87-45de-9c6a-5f7f6f285e21.png">

Reply:
You can check it  through
```
build/mvn dependency:tree -pl connector/connect/server
```

`protobuf-java-util` is the transitive dependency of `grpc-services`, but it is `runtime` scope.

```
[INFO] +- io.grpc:grpc-services:jar:1.47.0:compile
[INFO] |  \- com.google.protobuf:protobuf-java-util:jar:3.19.2:runtime
```

So I think we should explicit add this dependency for compilation

Reply:
Thanks, done. Updated the PR.